packages/razzle-plugin-scss/package.json

```diff
@@ -1,6 +1,24 @@
 {
   "name": "razzle-plugin-scss",
   "version": "2.2.0",
+  "description": "Use SCSS with Razzle",
   "main": "index.js",
-  "license": "MIT"
-}
+  "repository": "git@github.com:jaredpalmer/razzle.git",
+  "author": "Ray Andrew <raydreww@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest"
+  },
+  "dependencies": {
+    "autoprefixer": "^8.6.2",
+    "mini-css-extract-plugin": "^0.4.0",
+    "node-sass": "^4.9.0",
```

Comment:
Use `node-sass-chokidar`

Reply:
it's much less flaky.

Reply:
I want too, but [sass-loader](https://github.com/webpack-contrib/sass-loader) make it as peer dependencies and using node-sass API.

Are you perhaps successfully combine webpack and node-sass-chokidar together?

Reply:
Yeah it's just a warning. It works fine. We've used it for almost a year.